### PR TITLE
fixes #9935 - pass correct id to install errata for inc updates

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: app/lib/actions/katello/content_view/incremental_updates.rb [hunk 1]
```diff
@@ -40,9 +40,8 @@ def plan(version_environments, composite_version_environments, content, dep_solv
               handle_composites(old_new_version_map, composite_version_environments, output_for_version_ids, description, content[:puppet_module_ids])
             end
 
-            if systems.any? && !content[:errata_ids].blank?
-              errata_ids = ::Katello::Erratum.where(:uuid => content[:errata_ids]).pluck(:errata_id)
-              plan_action(::Actions::BulkAction, ::Actions::Katello::System::Erratum::ApplicableErrataInstall, systems, errata_ids)
+            if systems.any? && !content[:errata_ids].blank? #contnet[:errata_ids] are uuids
```
Comment (sthirugn) [inline app/lib/actions/katello/content_view/incremental_updates.rb]: s/contnet/content

**Comment by sthirugn** [comment]
Tested in recent compose and it worked fine.

**Comment by waldenraines** [comment]
Works fine for me and code looks good, APJ

**Comment by jlsherrill** [comment]
[test]

**Pull request merged**